docs/sql-ref-syntax-qry-select-like.md

```diff
@@ -21,12 +21,14 @@ license: |
 
 ### Description
 
-A LIKE predicate is used to search for a specific pattern.
+A LIKE predicate is used to search for a specific pattern. LIKE predicate also supports multiple patterns with quantifiers include `ANY`, `SOME` and `ALL`.
```

Comment:
> LIKE predicate also supports multiple patterns with quantifiers include `ANY`, `SOME` and `ALL`.

IMO we don't need to describe what the syntax is in the `Description` section and the statement in L32 and L51-53 is enough for users.

Reply:
Since Like All is a different syntax, I think it needs to be clearly introduced. Also refer to https://github.com/apache/spark/blob/master/docs/sql-ref-syntax-qry-select-groupby.md

Reply:
sgtm